core/src/main/java/org/elasticsearch/search/highlight/HighlighterParseElement.java

```diff
@@ -94,7 +94,7 @@ public void parse(XContentParser parser, SearchContext context) throws Exception
         }
     }
 
-    public SearchContextHighlight parse(XContentParser parser, QueryShardContext queryShardContext) throws IOException {
+    public static SearchContextHighlight parse(XContentParser parser, QueryShardContext queryShardContext) throws IOException {
```

Comment:
why did we need to make these methods static? I think I have missed where this is used